config/install/README.md

```diff
@@ -1,101 +1,591 @@
+
+```
+
+You can also view the subscription for information about the install:
+
+```bash
+kubectl get subscription -n kuadrant-system -o=yaml
+
 ```
 
+### Install the operand components
+
+Kuadrant has 2 additional operand components that it manages: `Authorino` that provides data plane auth integration and `Limitador` that provides data plane rate limiting. To set these up lets add a new `kustomization.yaml` in a new sub directory. We will re-use this later for further configuration. We do this as a separate step as we want to have the operators installed first.
 
+Add the following to your local directory.  For the purpose of this doc, we will use: `$KUADRANT_DIR/configure/kustomization.yaml`.
 
 ```bash
-kubectl get deployments -n gateway-system
+mkdir -p $KUADRANT_DIR/configure
+touch $KUADRANT_DIR/configure/kustomization.yaml
+
+```
+
+Add the following to the new kustomization.yaml:
 
 
-# NAME            READY   UP-TO-DATE   AVAILABLE   AGE
-# istiod          1/1     1            1           61s
-# sail-operator   1/1     1            1           81m
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - https://github.com/Kuadrant/kuadrant-operator//config/install/configure/standard?ref=v1.0.1 #change this version as needed (see https://github.com/Kuadrant/kuadrant-operator/releases)
+
 ```
 
-## Configure the installation
+Lets apply this to your cluster:
+
+```bash
+
+kubectl apply -k $KUADRANT_DIR/configure
 
-### TLS and DNS integration
+```
 
+### Verify kuadrant is installed and ready:
+
+```bash
+kubectl get kuadrant kuadrant -n kuadrant-system -o=wide
 
-Create the `$CLOUD_PROVIDER-credentials.env file` in the cloud provider directory `config/install/configure/$CLOUD_PROVIDER.` e.g. `aws-credentials.env` in the `config/install/configure/aws` directory. Apply the configuration for the desired cloud provider. Example AWS
+# NAME       STATUS   AGE
+# kuadrant   Ready    109s
 
 ```
-kubectl apply -k config/install/configure/aws
+
+You should see the condition with type `Ready` with a message of `kuadrant is ready`. 
+
+
+### Verify Istio is configured and ready:
+
+```bash
+kubectl get istio -n gateway-system
+
+#sample output
+# NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   VERSION   AGE
+# default   1           1       1        Healthy           v1.23.0   3d22h
 ```
 
-This will configure Kuadrant and Sail to install their components, set the credentials needed to access DNS zones in the cloud provider, and create a Let's Encrypt cluster issuer configured to use DNS-based validation.
 
-### Validate
 
-Validate Kuadrant is ready via the kuadrant resource status condition
+At this point Kuadrant is installed and ready to be used as is Istio as the gateway provider. This means AuthPolicy and RateLimitPolicy can now be configured and used to protect any Gateways you create. 
+
+
+## Configure DNS and TLS integration
```

Comment:
```suggestion
## (Optional) Configure DNS and TLS integration
```

To match the "(Optional) cert-manager" and "(Optional) Access to AWS" prereqs at the top, 
otherwise CRDs don't exist and no credentials.